src/main/ipc.ts

```diff
@@ -86,7 +86,7 @@ function configureProtocol(options: ElectronMainOptions): void {
             handleScope(options, data);
           }
 
-          callback('');
+          callback({ data: '', headers: { 'Access-Control-Allow-Origin': '*' } });
```

Comment:
Should we specifically allowlist the fetch urls we use?

Reply:
We could 404 for unknown URLs 🤔

Reply:
Fine with adding `404` in another PR. We can probably just debug log it out.